src/config/graphical-session/sway.md

```diff
@@ -0,0 +1,28 @@
+# Sway
+
+## Pre-installation
+
+You need a seat management program as a prerequisite. You can use either
+`elogind` or `seatd` by following the instructions
+[here](../session-management.md). Also [install a video
+driver](./graphics-drivers/index.md) such as `mesa-dri`.
+
+## Installation
+
+Install the `sway` package. You might also want to install `alacritty`, `dmenu`,
+`swaylock`, and `swayidle`, as they are programs used in in the default
+configuration file.
```

Comment:
This is entirely out of scope for Void docs.

(also for a out-of-the-box setup, you'd recommend `foot` :P )